index.html

```diff
@@ -0,0 +1,61 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8">
+    <title>Fork Me! FCC: Test Suite Template</title>
+    <link rel="stylesheet" href="css/style.css">
+  </head>
+
+  <body>
+    <style>
+      section{
+      display: flex;
+      }
+      @media only screen and (max-width: 600px) {
+        body {
+          background-color: lightblue;
+        }
+      }
+    </style>
+    <script src="https://cdn.freecodecamp.org/testable-projects-fcc/v1/bundle.js"></script>
+    <header id="header">
+      <img id="header-img" src="https://s3.amazonaws.com/freecodecamp/original_trombones.png" />
+     <nav id="nav-bar">
+       <ul>
+         <li><a class="nav-link" href="#feature">feature</a></li>
+         <li><a class="nav-link" href="#howitworks">How it Works</a></li>
+         <li><a class="nav-link" href="#pricing">Pricing</a></li>
+       </ul>
+
+     </nav>
+    
+    </header>
+    <section id="feature">
+      <iframe id="video" src="https://www.youtube.com/watch?v=HIj8wU_rGIU" frameborder="0" allow="autoplay encrypted-media" allowfullscreen></iframe>
+    </section>
+    <section id="howitworks">
+     <form id="form" action="https://wwww.freecodecamp.com/email-submit"> 
+       <input id="email" name="email" type="email" placeholder="Input your email" required> 
+       <input id="submit" class="btn" type="submit" value="submit">
+
+       
+        
+      </form>
+    </section>
+    <section id="pricing"></section> 
+    <!--
+
+    Hello Camper!
+
+    For now, the test suite only works in Chrome!
+    Please read the README below in the JS Editor before beginning.
+    Feel free to delete this message once you have read it.
+    Good luck and Happy Coding!
+
+    - The freeCodeCamp Team
+
+    -->
```

Comment:
Please remove unnecessary comments in your codebase. This makes code your unclean.

Reply:
- Good!